llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -721,6 +698,42 @@ bool isTargetCompatibleWithModule(const std::optional<std::string> &Target,
   return true;
 }
 
+// @param OutTables List of tables (one for each target) to output results
```

Comment:
What is the reason for moving this function? If there are no strong reasons, then I would prefer to have it in place to reduce the diff

Reply:
It needed some functions defined later, so I moved it down. I moved it back but added the necessary forward declarations